clients/cpp-console/src/common/mainAudio.cpp

```diff
@@ -71,7 +71,12 @@ void log_t(T v, Args... args)
 	std::chrono::system_clock::time_point now = chrono::system_clock::now();
 	std::time_t now_c = std::chrono::system_clock::to_time_t(now);
 	std::tm now_tm;
-	localtime_s(&now_tm, &now_c);
+#ifdef LINUX
+    localtime_r(&now_c, &now_tm);
```

Comment:
Align number of spaces at the beginning to match the rest of the file (do not use tabs, just spaces)

Reply:
Fixed.